HandParam[] hand_param
int32 force_mode
int32 hand_index
int32 hand_type